fineract-provider/src/main/resources/META-INF/persistence.xml

```diff
@@ -32,6 +32,7 @@
             <property name="openjpa.jdbc.DBDictionary" value="org.apache.fineract.infrastructure.core.domain.MySQLDictionaryCustom"/>
             <!--<property name="openjpa.InverseManager" value="true(Action=warn)"/> -->
             <property name="openjpa.Log" value="slf4j"/>
+            <!-- <property name="openjpa.Log" value="DefaultLevel=WARN, Runtime=INFO, Tool=INFO, SQL=TRACE"/> -->
```

Comment:
we don't need migrations? for the subInstallment number??

Reply:
Yes we do. fixing...